examples/tutorial_examples/plot_optic_radiations.py

```diff
@@ -1,6 +1,6 @@
 """
 =============================
-Plotting the Optic Radiations
+Adding new bundles into pyAFQ (Optic Radiations Example)
```

Comment:
I don't think that you intended to include changes in this file?

Reply:
he meant some of them. I will clean this up